packages/react/.storybook/main.ts

```diff
@@ -92,9 +89,10 @@ module.exports = {
   },
   framework: '@storybook/react',
   stories,
-  webpack(config) {
-    const babelLoader = config.module.rules.find((rule) => {
-      return rule.use.some(({ loader }) => {
+  // TODO: Fix typings for this function
+  webpack(config: any) {
```

Comment:
If statically typing this is desired, there are two options:

1. Only use `Configuration` type from Webpack

```ts
import type { Configuration } from 'webpack';
...
webpack(config: Configuration) {
```

2. Type the entire config using `StorybookConfig`

```ts
import type { StorybookConfig } from '@storybook/core-common';

const config: StorybookConfig = { /** */ }

module.exports = config;
```

Reply:
Yep nice call, there was a few other changes needed in the body of the config to account for the union types used in the `webpackFinal` prop. The use of the typings also caught the fact that the `webpack` property should have actually been renamed to `webpackFinal` according to the storybook 6.5 docs